src/components/Card.astro

```diff
@@ -0,0 +1,51 @@
+---
+const { colour, title, description } = Astro.props;
+---
+
+<style define:vars={{ colour }}>
+* {
+  font-family: Helvetica, sans-serif;
+}
+
+div {
+  background-color: var(--colour);
+  border-radius: 0.3em;
+  width: 30em;
+  height: 15em;
+}
+
+h1, p, button {
+  position: relative;
```

Comment:
Instead of using position relative and super positioning everything using top, left, and right units i'd recommend using margin and padding. You'll save a lot of lines by having a padding of 1 rem.

Reply:
Very good point! I will update it to use margins/padding instead of positioning.